Generated/PriceView.m

```diff
@@ -6,10 +6,12 @@ - (instancetype)initWithStyle:(UITableViewCellStyle)style reuseIdentifier:(NSStr
     if(self)
     {
 	_backgroundView = [UIView new];
+	_backgroundView.layer.cornerRadius = 0.8533333333333334;
```

Comment:
@csacsi Good point, although this is the actual corner radius in the Sketch file (edge case), in a normal use case this won't happen